CHANGELOG.md

```diff
@@ -38,6 +38,12 @@
 
 * `skbio.stats.composition` now has methods to compute additive log-ratio transformation and inverse additive log-ratio transformation (`alr`, `alr_inv`) as well as a method to build a basis from a sequential binary partition (`sbp_basis`).
 
+* Added option to return a capture group compiled regex pattern to any class inheriting ``GrammaredSequence`` through the ``to_regex`` method. ([#1431](https://github.com/biocore/scikit-bio/issues/1431))
+
+* Added `Dissimilarity.within` and `.between` to obtain the respective distances and express them as a `DataFrame`.
+
+* Added Kendall Tau as possible correlation method in the `skbio.stats.distance.mantel` function ([#1675](https://github.com/biocore/scikit-bio/issues/1675)).
```

Comment:
@AnnaTruzzi I just noticed that this entry is not in the most recent section of the CHANGELOG.md file. Would you mind moving this bullet point after line 9?